config/config.go

```diff
@@ -910,6 +910,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.beachfront.endpoint", "https://display.bfmio.com/prebid_display")
 	v.SetDefault("adapters.beachfront.extra_info", "{\"video_endpoint\":\"https://reachms.bfmio.com/bid.json?exchange_id\"}")
 	v.SetDefault("adapters.beintoo.endpoint", "https://ib.beintoo.com/um")
+	v.SetDefault("adapters.between.endpoint", "http://{{.Host}}/s2s/")
```

Comment:
Why do you need publishers to provide the domain? PBS does not act as a general proxy. I realize a few other adapters do this today, but it's something we're still working with bidders to fix.

Reply:
I see you marked this resolved, but I didn't see your answer. I might have missed a reply in another comment thread.